apollo-federation/src/sources/connect/validation/extended_type.rs

```diff
@@ -288,7 +318,7 @@ fn validate_field(
                     })
                 }
             }
-        } else if let Some((template, coordinate)) = url_template {
+        } else if let Some((template, coordinate)) = url_template.clone() {
```

Comment:
Does this need to be cloned? Would Option::as_ref work otherwise?

Reply:
reverted this